lib/ssl/src/tls_client_ticket_store.erl

```diff
@@ -103,6 +103,7 @@ update_ticket(Key, Pos) ->
 
 init(Args) ->
     process_flag(trap_exit, true),
+    proc_lib:set_label(tls_13_client_session_tickets),
```

Comment:
Remove, processes with a registered name don't need labels.
And that goes for all registered processes.

Reply:
Ah, good catch missed that!